catalyst_voices/apps/voices/lib/pages/account/unlock_keychain_dialog.dart

```diff
@@ -140,12 +141,14 @@ class _UnlockPasswordPanel extends StatelessWidget {
 }
 
 class _UnlockPassword extends StatelessWidget {
+  final VoidCallback onUnlock;
   final TextEditingController controller;
   final LocalizedException? error;
```

Comment:
Usually callbacks go below, both in the field list and and the constructor. Also constructors often have their params in the same order as the field list

```suggestion
  final TextEditingController controller;
  final LocalizedException? error;
  final VoidCallback onUnlock;
```